lib/PasswordStrategy.js

```diff
@@ -48,7 +48,8 @@ function Strategy(options) {
             sysPassword: password
           }, function(err, verified) {
             if(err) {
-              return callback(err);
+              // FIXME: Ignore errors to ensure all IDs are checked
+              return callback();
```

Comment:
This doesn't seem right. An error should not be thrown when a password is invalid, it should result in `verified` being `false` -- is that not the case here? What error was being thrown?

An error should indicate that something exceptional happened (vs. just a password being incorrect) and it should rightly abort, IMO. If an error is being thrown due to an invalid password, we should probably fix the problem there so there is no error but `verified` is `false`.